git_tools/tag_all_repos.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+"""
+*****************************************************************
+Licensed Materials - Property of IBM
+(C) Copyright IBM Corp. 2020. All Rights Reserved.
+US Government Users Restricted Rights - Use, duplication or
+disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+*****************************************************************
+
+*******************************************************************************
+Script: tag_all_repos.py
+A script that can be used to create the same annotated git tag in all
+repos within an organization.
+*******************************************************************************
+"""
+
+import argparse
+import os
+import sys
+import pathlib
+import yaml
+import requests
+
+sys.path.append(os.path.join(pathlib.Path(__file__).parent.absolute(), '..', 'open-ce'))
+import utils # pylint: disable=wrong-import-position
+
+def _make_parser():
+    ''' Parser input arguments '''
+    parser = argparse.ArgumentParser(
+        description = 'Tag all repos in an organization.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument(
+        'github_org',
+        type=str,
```

Comment:
Should we have a default value here?

Reply:
I kinda like having to be explicit with it, so there's less chance of somebody making random tags across an org.